src/structure/UDPSocket.ts

```diff
@@ -93,6 +95,15 @@ class UDPSocket {
 			this.socket.on('message', messageHandler);
 			this.socket.on('error', errorHandler);
 			this.socket.on('close', closeHandler);
+
+			if(this.timeout != 0)
```

Comment:
Spacing, comparison operators, and bracket placement should follow linting rules